dist/etc/conf/samr21-xpro.env

```diff
@@ -0,0 +1,77 @@
+HIL_DUT_RST_GPIO?=
+HIL_DUT_RST_PORT?=
+HIL_DUT_RST_PIN?=
+
+HIL_DUT_CTS_GPIO?=
+HIL_DUT_CTS_PORT?=
+HIL_DUT_CTS_PIN?=
+
+HIL_DUT_RTS_GPIO?=
+HIL_DUT_RTS_PORT?=
+HIL_DUT_RTS_PIN?=
+
+HIL_DUT_RX_GPIO?="PB23/EXT1-10"
+HIL_DUT_RX_PORT?=1
+HIL_DUT_RX_PIN?=23
+
+HIL_DUT_TX_GPIO?="PB22/EXT1-9"
+HIL_DUT_TX_PORT?=1
+HIL_DUT_TX_PIN?=22
+
+HIL_DUT_NSS_GPIO?="PB3/EXT1-15"
+HIL_DUT_NSS_PORT?=1
+HIL_DUT_NSS_PIN?=3
+
+HIL_DUT_SCK_GPIO?="PB23/EXT1-18"
+HIL_DUT_SCK_PORT?=1
+HIL_DUT_SCK_PIN?=23
+
+HIL_DUT_MISO_GPIO?="PB2/EXT1-17"
+HIL_DUT_MISO_PORT?=1
+HIL_DUT_MISO_PIN?=2
+
+HIL_DUT_MOSI_GPIO?="PB22/EXT1-16"
+HIL_DUT_MOSI_PORT?=1
+HIL_DUT_MOSI_PIN?=22
+
+HIL_DUT_SCL_GPIO?="PA17/EXT1-12"
+HIL_DUT_SCL_PORT?=0
+HIL_DUT_SCL_PIN?=17
+
+HIL_DUT_SDA_GPIO?="PA16/EXT1-11"
+HIL_DUT_SDA_PORT?=0
+HIL_DUT_SDA_PIN?=16
+
+HIL_DUT_PWM_GPIO?=
+HIL_DUT_PWM_PORT?=
+HIL_DUT_PWM_PIN?=
+
+HIL_DUT_DAC_GPIO?="PA6/EXT1-3"
+HIL_DUT_DAC_PORT?=0
+HIL_DUT_DAC_PIN?=6
+
+HIL_DUT_ADC_GPIO?=
+HIL_DUT_ADC_PORT?=
+HIL_DUT_ADC_PIN?=
+
+HIL_DUT_IC_GPIO?=
+HIL_DUT_IC_PORT?=
+HIL_DUT_IC_PIN?=
+
+HIL_DUT_GPIO0_GPIO?="PA19/EXT1-8"
+HIL_DUT_GPIO0_PORT?=0
+HIL_DUT_GPIO0_PIN?=19
```

Comment:
current config in Jenkins is debug0=pa13, debug1=pa18, and debug2=pa19 - and debug2 is okay on master but here all 3 fail now.

Reply:
Thanks, updated, we will see the CI

Reply:
It matches master now.  Anything else?

Reply:
nope, at least if you haven't mis-copied other pin configs to 😬